robot/web/web.go

```diff
@@ -89,17 +89,18 @@ type Service interface {
 type webService struct {
 	resource.Named
 
-	mu        sync.Mutex
-	r         robot.Robot
-	rpcServer rpc.Server
-	modServer rpc.Server
+	mu            sync.Mutex
+	r             robot.Robot
+	rpcServer     rpc.Server
+	unixModServer rpc.Server
```

Comment:
design thing from my first draft -- concerned about locking / concurrent touching of data from having two module servers now

may not be a problem but we should at least think about it. I'll catch up with folks IRL